io.typora.editor.json

```diff
@@ -0,0 +1,74 @@
+{
+  "id": "io.typora.editor",
+  "command": "/app/start.sh",
+  "separate-locales": false,
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "20.08",
+  "sdk": "org.freedesktop.Sdk",
+  "finish-args": [
+    "--device=dri",
+    "--filesystem=host",
+    "--share=ipc",
+    "--socket=x11"
+  ],
+  "modules": [
+    {
+      "name": "typora",
+      "modules": [
+        {
+          "name": "pandoc",
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://github.com/jgm/pandoc/releases/download/2.11.2/pandoc-2.11.2-linux-amd64.tar.gz",
+              "sha256": "25e4055db5144289dc45e7c5fb3616ea5cf75f460eba337b65474d9fbc40c0fb"
+            }
+          ],
+          "buildsystem": "simple",
+          "build-commands": [
+            "install -Dm755 bin/pandoc /app/bin/pandoc"
+          ]
+        }
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://typora.io/linux/Typora-linux-x64-0.9.96.tar.gz",
```

Comment:
If the apps license doesn't allow redistribution (what is likely the case), you should use extra-data type source for it.

Reply:
I forgot to change the project_license property, what value do you think is more appropriate?
However, I think he just copied a EULA licence text from somewhere else and put it on his website.
This link [https://support.typora.io/License-Agreement/](https://support.typora.io/License-Agreement/) mentions several times the use of a license key that doesn't exist, there's no license or addons to buy in order to use this application.
Also, point #4.7 says: "Exception: repackage (provide installation packages/scripts) Typora on Linux system is allowed if all contents (includes all binaries, resources files, scripts, etc) of Typora are not modified or deleted."
So repackaging into a flatpak should be fine by this license.

Reply:
Yeah, that's fine.